jep/0000/README.adoc

```diff
@@ -0,0 +1,358 @@
+
+.Metadata
+[cols="1h,1"]
+|===
+| JEP
+| 0000
+
+| Title
+| End Java 8 support in Jenkins
+
+| Sponsor
+| link:https://github.com/MarkEWaite[Mark Waite]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Process
+
+| Created
+| 2022-01-28
+
+| BDFL-Delegate
+| TBD
+
+| Pull requests
+a|
+
+- link:https://github.com/jenkinsci/jenkins/pull/6083[PR 6083] - Drop Java 8 from Jenkins core
+- link:https://github.com/jenkinsci/jenkins/pull/6086[PR 6086] - Switch Jenkins acceptance test harness in core to Java 11
+- link:https://github.com/jenkinsci/jenkins/pull/6092[PR 6092] - Add administrative monitor to remind of upcoming end of Java 8 support
+- link:https://github.com/jenkinsci/acceptance-test-harness/pull/726[PR 726] - Drop Java 8 from acceptance test harness
+
+| Discussions-To
+| link:https://groups.google.com/g/jenkinsci-dev[Jenkins Developer Mailing List]
+
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+Java 11 was released on September 25, 2018.
+It is an LTS version with a long support timeline.
+Java 8 was released in March 2014.
+The link:https://www.oracle.com/java/technologies/java-se-support-roadmap.html[Oracle Java SE Support Roadmap] states that premier support for Java 8 ends in March 2022.
+Java 8 enhancements have stopped as effort focuses on Java 11, Java 17, and beyond.
+
+This Jenkins Enhancement Proposal describes actions required
+to make Jenkins end its support of Java 8.
+
+== Specification
+
+The specification has been created as a result of a link:https://groups.google.com/g/jenkinsci-dev/c/YghQ0YP4m78/m/LO9AFa_GAgAJ[Jenkins developers mailing list discussion].
+
+All stories in this JEP have tickets created in Jira or pull requests in GitHub.
+Jira tickets are aggregated in
+link:https://issues.jenkins.io/browse/JENKINS-67688[JENKINS-67688].
+
+If the tickets are not linked in the description, they can be found there.
+
+=== Goals and non-goals
+
+Goals:
+
+* Jenkins core development requires Java 11 or newer
+* Jenkins plugin development requires Java 11 or newer when the plugin updates its minimum Java version to a Jenkins core version that does not support Java 8
+
+Non-goals:
+
+* Building all components with Java 11
+* Jenkins replace older API calls with Java 11 API calls
+** Jakarta EE upgrade is not required as part of this JEP
+* All plugins are operational with Java 11
+** Some Jenkins components may be updated to support features offered in Java 11 and beyond, but there is no plan to update all tools
```

Comment:
Which “components” and “tools” are being discussed here? Is this deliberately a sub-bullet of the one discussing plugins?